packages/quantic/force-app/main/default/lwc/quanticUserAction/templates/viewAction.html

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="slds-grid">
+        <div class="slds-grid slds-grid_align-center user-action__icon-container slds-var-p-right_x-small">
+            <div class="slds-var-p-vertical_xx-small">
+                <lightning-icon class={iconClass} icon-name={iconName}
+                    size="xx-small"></lightning-icon>
+            </div>
+
+            <div class="slds-var-p-vertical_xx-small">
+                <div class="user-action__separator"></div>
+            </div>
+        </div>
+        <div class="slds-col">
+            <div class="user-action__title slds-var-p-vertical_x-small slds-text-title">
+                <a href={contentIdValue} target="_blank">
+                    {actionTitle}
+                </a>
```

Comment:
Question here: why not using a getter to check if the action is a VIEW action, that way you could eliminate the need for templates and extra css files? I think a getter would remove the need for a bit of duplication what do you think?

Was it in the case where we add more cases of userAction types in the future or something?

Reply:
It's more like a preference imo, both have pros and cons and both would work.

But yes it's likely to have more cases that need a specific treatment, for example the CLICK action could become a link as well, and the href of that link would not be `contentIdValue` like it is the case for view events, it would be `clickUri` so if use one template it can become complicated  as we will have conditions or getters that give us information about whether we need to display links or not other getters to provide information about which value to use as the href.

so considering this I found that from a readability perspective it was easier to see these action in different templates comparing to seeing them in one template but that contain a lot of conditional rendering.